server/channel.js

```diff
@@ -0,0 +1,72 @@
+var WebSocketServer = require("ws").Server;
+var fs              = require("fs");
+
+// this channel.js is the server-side abstraction over websockets
```

Comment:
I've made some additional changes, the new combined Channel file is less LOC than the original client or server side channel, not sure how. It's working great anyhow!